dash/_callback.py

```diff
@@ -40,9 +40,11 @@ def to_plotly_json(self):  # pylint: disable=no-self-use
 
     @staticmethod
     def is_no_update(obj):
-        return isinstance(obj, NoUpdate) or obj == {
-            "_dash_no_update": "_dash_no_update"
-        }
+        return (
+            isinstance(obj, NoUpdate)
+            or isinstance(obj, dict)
+            and obj == {"_dash_no_update": "_dash_no_update"}
```

Comment:
```suggestion
            or (isinstance(obj, dict)
            and obj == {"_dash_no_update": "_dash_no_update"})
```
that's not going to pass linting... but the point is it's hard to read if I have to try and remember (or learn) that `and` has higher precedence than `or`